hudi-common/src/main/java/org/apache/hudi/avro/MercifulJsonConverter.java

```diff
@@ -286,7 +284,7 @@ private static JsonToAvroFieldProcessor generateArrayTypeHandler() {
       public Pair<Boolean, Object> convert(Object value, String name, Schema schema)
           throws HoodieJsonToAvroConversionException {
         Schema elementSchema = schema.getElementType();
-        List listRes = new ArrayList();
+        List listRes = new ArrayList<>();
```

Comment:
List<Object> listRes = new ArrayList<>(); ?

Reply:
How about List<?> listRes = new ArrayList<>();

Reply:
makes sense this is Jackson API